cli/module_generate/scripts/generate_stubs.py

```diff
@@ -3,15 +3,92 @@
 import subprocess
 import sys
 from importlib import import_module
+from typing import List, Set
 
 
-def return_attribute(resource_name: str, attr: str) -> ast.Attribute:
+def return_attribute(value: str, attr: str) -> ast.Attribute:
```

Comment:
That was more of a function design decision, since this function doesn't technically know it's a `resource_name`. It's taking a value and assigning it; we're just using it as a way to set the `resource_name` as the value.